README.md

```diff
@@ -93,13 +93,15 @@ Wechaty already held lots of talk and got a lot of blogs in the past 4 years, he
 
 ```javascript
 
-import { WechatyBuilder } from 'wechaty'
+import { WechatyBuilder } from "wechaty"
```

Comment:
Please use `'` instead of `"` so that your modification still follows the Wechaty coding style.

Reply:
Fixed